### ISSUE TITLE
Web (online) debugger is completely unresponsive while consuming 100% CPU (1 core)

### ISSUE THREAD

**Issue by efogdev** [oven-sh/bun#15715]
### What version of Bun is running?

1.1.38

### What platform is your computer?

Linux 6.12.4-1-cachyos x86_64 unknown 

### What steps can reproduce the bug?

1. Start app with `bun --inspect=0.0.0.0:13010/debug --watch src/index.ts`
2. Open `debug.bun.sh/#localhost:13010/debug`

### What is the expected behavior?

Actual debugger is working at least a little

### What do you see instead?

Debugger UI is stuck in "console" tab and completely unresponsive, probably in some infinite loop

### Additional information

Comprehensive 1-minute video: https://www.youtube.com/watch?v=EGJ5a-5caX8
The browser is clean installation of Chromium (also tried in my Vivaldi, same result)
It's driving me nuts because I can't build the debugger because it's really far off my tech stack and I don't have Mac
I'm under GNOME 47 Wayland, OS is arch linux, fully updated

**Comment by RiskyMH** [comment]
@efogdev can you try my fix (in canary at moment) and see if it improves the debugger for you.

```zig
bun upgrade --canary
```

**Comment by efogdev** [comment]
No changes — once I open the debugger, its completely unresponsive

**Comment by efogdev** [comment]
@RiskyMH any chances to get a build? Ideally not minified, uglified etc. So I can deploy it locally and debug the debugger properly

**Comment by divmgl** [comment]
I'm running into this issue now. Did you ever figure it out?

**Comment by RiskyMH** [comment]
No, on windows it was fine for me, and I haven't tried a Linux browser (I did try it as running bun tho)

I think there is a chance one thing isnt getting received right and thus aimlessly waiting or something odd.

**Comment by RiskyMH** [comment]
Actually I do remember seeing it "not connect" sometimes, but that was always fixable by typing in the console input.

I don't know why that's needed or the other issue reported here (I see console was typed in this video)
